homeassistant/components/reolink/host.py

```diff
@@ -138,7 +138,28 @@ async def update_states(self) -> None:
 
     async def disconnect(self):
         """Disconnect from the API, so the connection will be released."""
-        await self._api.unsubscribe()
+        try:
+            await self._api.unsubscribe()
+        except aiohttp.ClientConnectorError as err:
```

Comment:
Consider joining the try... except blocks and do both calls inside the same block. Maybe we don't need to have specific logging but just a general logging for `disconnect`?

Reply:
I did join all the errors into once except.
However I would like to have seperate try excepts for unsubscribing and logout, because if we fail to unsubscribe I would still like to try to logout. (other way around is not possible because we need the connection to still be allive for the unsubscription).